packages/plugin-flex/src/commands/flex/plugins/describe/release.ts

```diff
@@ -82,6 +88,9 @@ export default class FlexPluginsDescribeRelease extends InformationFlexPlugin<De
    */
   /* istanbul ignore next */
   get _flags(): OutputFlags<typeof FlexPluginsDescribeRelease.flags> {
-    return this.parse(FlexPluginsDescribeRelease).flags;
+    if (!this._parsedFlags) {
+      throw new TwilioCliError('Flags are not parsed yet');
+    }
+    return this._parsedFlags;
```

Comment:
Why are we changing `_flags` to `_parsedFlags`?

Reply:
We are not changing. Both of them have different roles. _flags is a getter method that was previously there and used throughout the class. _parsedFlags is the one that stores the parsed flags after the parse fn asynchronously returns it. We are able to throw an error if the parse is not done yet, with this approach

Reply:
I still don't understand this - if `init` _has_ to be called at first, then store the value in `_flags` and the getter just returns that value and don't call the `parse` like it was doing before

Reply:
How can the getter and the variable where we store be both `_flags`? We are doing exactly what you said except that it is stored in `_parsedFlags` and the getter just returns it

Reply:
Tracked as part of another comment, hence resolving